src/app/demo/demo.component.spec.ts

```diff
@@ -23,4 +26,341 @@ describe("DemoComponent", () => {
   it("should create", () => {
     expect(component).toBeTruthy();
   });
+
+  it(`should have as title 'Title'`, () => {
+    expect(component.slots.title).toEqual("Title");
+  });
+
+  it(`should have as subtitle 'SubTitle'`, () => {
+    expect(component.slots.subtitle).toEqual("Subtitle");
+  });
+
+  it(`should have as page title 'ReadAlong Studio'`, () => {
+    expect(component.getTitle()).toEqual("ReadAlong Studio");
+  });
+
+});
+
+// ================test page title=================
+// === test editable test page before edited ===
+describe("DemoComponent-pagetitle-defalutvalue", () => {
+  let component: DemoComponent;
+  let fixture: ComponentFixture<DemoComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [MaterialModule],
+      declarations: [DemoComponent],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA],
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(DemoComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it("test editable defalut page title ", () => {
+     const { debugElement } = fixture;
+     const title1 = debugElement.query(By.css('#spanid')).nativeElement;
+     expect(title1.textContent).toContain('ReadAlong Studio');
+  });
+});
+
+// === test editable test page after edited ===
+describe("DemoComponent-pagetitle-edittvalue", () => {
+  let component: DemoComponent;
```

Comment:
Please put all the tests for this DemoComponent into one `describe` function for readability.